delta-app/src/main/java/io/cdap/delta/app/service/AssessmentHandler.java

```diff
@@ -309,7 +313,15 @@ private void respond(String namespaceName, HttpServiceResponder responder, Names
     }
 
     try {
-      endpoint.respond(new DraftService(context, new SystemServicePropertyEvaluator(context)), namespace);
```

Comment:
Can we `setEscaling('false')` here itself - https://github.com/data-integrations/delta/blob/develop/delta-app/src/main/java/io/cdap/delta/store/SystemServicePropertyEvaluator.java#L32

Reply:
good catch :). I was thinking SystemServicePropertyEvaluator was in cdap core.